### PR TITLE
[graphql/rpc]add epoch doc comments

### PR DESCRIPTION
**Pull request by stefan-mysten**
## Description 

Adds doc comments for the `epoch` and `validatorSet` types.

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

**Review by wlmyng — APPROVED** [review]

File: crates/sui-graphql-rpc/schema/current_progress_schema.graphql [hunk 1]
```diff
@@ -202,13 +202,37 @@ type EndOfEpochTransaction {
 }
 
 type Epoch {
+	"""
+	The epoch's id
```
Comment (amnn) [inline crates/sui-graphql-rpc/schema/current_progress_schema.graphql:206]: Explain that this is a sequence number? (i.e. the first epoch is epoch 0, and it increases one at a time).

File: crates/sui-graphql-rpc/schema/current_progress_schema.graphql [hunk 2]
```diff
@@ -202,13 +202,37 @@ type EndOfEpochTransaction {
...
+	"""
 	epochId: Int!
+	"""
+	The reference gas price for this epoch
```
Comment (amnn) [inline crates/sui-graphql-rpc/schema/current_progress_schema.graphql:210]: Can you copy the explanation I added for the reference gas price in the system state summary here?

**Review by amnn — APPROVED** [review]
Some extra clarifications, but looks good!

File: crates/sui-graphql-rpc/schema/current_progress_schema.graphql (hunk 2 above)
Comment (stefan-mysten) [reply crates/sui-graphql-rpc/schema/current_progress_schema.graphql:210]: Totally, thanks for suggesting it.

**Pull request merged**